roles/openshift_web_console/tasks/install.yml

```diff
@@ -71,6 +71,9 @@
     - set_fact:
         config_to_migrate: "{{ master_config_output.content | b64decode | from_yaml }}"
 
+    - set_fact:
+        cro_plugin_enabled: "{{ config_to_migrate.admissionConfig is defined and config_to_migrate.admissionConfig.pluginConfig is defined and config_to_migrate.admissionConfig.pluginConfig.ClusterResourceOverrides is defined }}"
```

Comment:
@sdodson Let me know if there's a better way to check if a nested key is defined.

Reply:
I think we'd considered writing a filter that would handle this more cleanly but I'm not finding such a filter. So this is pretty common across the code base.